README.md

```diff
@@ -48,18 +54,17 @@ then in your `glance.yml` config file, add the following:
 | `same-tab`      | Open the URL in the same tab. Value customizable per container                                                           | `false`             |
 | `ignore-status` | Status of the containers will not be displayed                                                                           | `false`             |
 
-## Configuration
+## Configuration (required)
 
 Then, for every container you want to monitor, add the following labels to its compose file:
```

Comment:
Yeah, configuration is required if you want to make widget appear in the homepage; in fact the sentence below declares that "for every container you want to monitor" you need to add some labels.

I'll pass on this